src/utils/interface.js

```diff
@@ -8,6 +8,15 @@ const pkg = require(path.resolve(__dirname, '../../package.json'));
 const conf = new Configstore(pkg.name);
 const themeColor = conf.get('themeColor');
 
+let themeCommon = (bottom, height) => {
+  return {
+    border: false,
+    bottom: bottom,
+    height: height,
+    width: '100%'
+  }
+}
+
```

Comment:
@sog01 Can we just define an object containing the common values for now and declare it as `baseStyles`

We can spread the `baseStyles` object into the other blessed objects in their respective objects.

I feel that if we create a new function, the code becomes a little more complex and harder to follow